server/helpers/ffmpeg-utils.ts

```diff
@@ -124,36 +125,71 @@ function getDurationFromVideoFile (path: string) {
   })
 }
 
-async function generateImageFromVideoFile (fromPath: string, folder: string, imageName: string, size: { width: number, height: number }) {
-  const pendingImageName = 'pending-' + imageName
+async function generateImageFromVideoFile (
+  fromPath: string,
+  folder: string,
+  imageName: string,
+  size: {
+    width: number,
+    height: number
+  },
+  count = 1 // count > 1 generates thumbnails and their vtt file
```

Comment:
I think it would be better to have a dedicated function instead of adding this parameter.